src/components/waitlist/WaitlistPanel.tsx

```
@@ -0,0 +1,204 @@
+                  </div>
+                ))}
+              </div>
+            </div>
+          )}
+
+          {/* Info */}
+          <div className="bg-blue-50 p-3 rounded">
+            <p className="font-mono text-[10px] text-blue-700">
+              <strong>How it works:</strong> When a spot opens up, the next person
+              on the waitlist receives a notification and has 60 minutes to confirm
+              their RSVP. If they don&apos;t confirm, the next person is promoted.
```

ESLint

[error] Replace `⏎··············on·the·waitlist·receives·a·notification·and·has·60·minutes·to·confirm⏎··············their·RSVP.·If·they·don&apos;t` with `·on·the·waitlist⏎··············receives·a·notification·and·has·60·minutes·to·confirm·their·RSVP.·If·they·don&apos;t⏎·············`

(prettier/prettier)